doc/doxyfile.in

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = CSFML
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
+PROJECT_NUMBER         = @PROJECT_VERSION_MAJOR@
```

Comment:
Shouldn't this be `${PROJECT_VERSION}`?

Reply:
Yeah I think so. Not sure why I'd drop the minor and patch numbers here.